src/test/java/com/specialwarriors/conal/UserServiceTest.java

```diff
@@ -0,0 +1,111 @@
+package com.specialwarriors.conal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import com.specialwarriors.conal.common.auth.oauth.GithubOAuth2WebClient;
+import com.specialwarriors.conal.common.exception.GeneralException;
+import com.specialwarriors.conal.user.domain.User;
+import com.specialwarriors.conal.user.repository.UserRepository;
+import com.specialwarriors.conal.user.service.UserService;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.data.redis.core.ValueOperations;
+
+@ExtendWith(MockitoExtension.class)
+public class UserServiceTest {
+
+    @Mock
+    private UserRepository userRepository;
+
+    @Mock
+    private RedisTemplate<String, String> redisTemplate;
+
+    @Mock
+    private GithubOAuth2WebClient githubOAuth2WebClient;
+
+    @InjectMocks
+    private UserService userService;
+
+    private User mockUser;
+
+    @BeforeEach
+    void setUp() {
+        mockUser = new User(1, "홍길동", "fdsf");
+    }
+
+    @Test
+    @DisplayName("유저 아이디로 사용자를 조회할 수 있다")
+    void findUserByUserId() {
+
+        // given
+        given(userRepository.findById(1L)).willReturn(Optional.of(mockUser));
+
+        // when
+        User result = userService.getUserByUserId(1L);
+
+        // then
+        assertThat(result).isEqualTo(mockUser);
+        verify(userRepository).findById(1L);
+    }
```

Comment:
```suggestion
    @Test
    @DisplayName("유저 아이디로 사용자를 조회할 수 있다")
    void findUserByUserId() {

        // given
        long userId = 1L;
        given(userRepository.findById(userId)).willReturn(Optional.of(mockUser));

        // when
        User result = userService.getUserByUserId(userId);

        // then
        assertThat(result).isEqualTo(mockUser);
        verify(userRepository).findById(userId);
    }
```
의미의 명확성을 위해 상수값보다 적절한 변수를 이용하는 것이 어떨까요?